src/components/Events/PreviewEvent.js

```diff
@@ -1,32 +1,52 @@
 import React from 'react';
+import { putUpdatedDraft } from '../../actions/Edit-Draft';
 import { postNewEvent, resetNewEventState } from '../../actions/New-Event';
 
 
 export default function PreviewEvent (props) {
 
   function onSubmit() {
-    const newEvent = {
+    const event = {
       userId: props.currentUser.id,
       title: props.eventState.title,
       draft: false,
       description: props.eventState.description,
+      city: props.eventState.city,
```

Comment:
We don't need these anymore, since theyre stored in locationCity: {city: '', state: ''}